CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [16.1.1] - TBD
+### Changed
+* Changed version of `hive.version` to `2.3.7` (was `2.3.2`).
```

Comment:
```suggestion
* Changed version of `hive.version` to `2.3.7` (was `2.3.2`). This allows Circus Train to be used on JDK>=9.
```
(let's add an extra bit of detail here so public end users can see what the main benefit of this change is).